File: server/database/entities/Sport.php
<?php

class Sport {
    private $id;
    private $name;
    private $description;
    private $minPlayers;
    private $maxPlayers;
    
    public function getId() {
        return $this->id;
    }

    public function getName() {
        return $this->name;
    }

    public function getDescription() {
        return $this->description;
    }

    public function getMinPlayers() {
        return $this->minPlayers;
    }

    public function getMaxPlayers() {
        return $this->maxPlayers;
    }

    public function setId($id): void {
        $this->id = $id;
    }

    public function setName($name): void {
        $this->name = $name;
    }

    public function setDescription($description): void {
        $this->description = $description;
    }

    public function setMinPlayers($minPlayers): void {
        $this->minPlayers = $minPlayers;
    }

    public function setMaxPlayers($maxPlayers): void {
        $this->maxPlayers = $maxPlayers;
    }
    
    public static function fromObject($obj) {
        $sport = new Sport();
        if (isset($obj->id)) {
            $sport->id = $obj->id;
        }
        if (isset($obj->name)) {
            $sport->name = $obj->name;
        }
        if (isset($obj->description)) {
            $sport->description = $obj->description;
        }
        if (isset($obj->minPlayers)) {
            $sport->minPlayers = $obj->minPlayers;
        }
        if (isset($obj->maxPlayers)) {
            $sport->maxPlayers = $obj->maxPlayers;
        }
        return $sport;
    }

}

File: client/model/src/main/java/com/example/model/Sport.java
package com.example.model;

/**
 * The type Sport.
 */
public class Sport {

    private int id;
    private String name;
    private String description;
    private int minPlayers;
    private int maxPlayers;

    /**
     * Gets id.
     *
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * Sets id.
     *
     * @param id the id
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * Gets name.
     *
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * Sets name.
     *
     * @param name the name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * Gets description.
     *
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * Sets description.
     *
     * @param description the description
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * Gets min players.
     *
     * @return the min players
     */
    public int getMinPlayers() {
        return minPlayers;
    }

    /**
     * Sets min players.
     *
     * @param minPlayers the min players
     */
    public void setMinPlayers(int minPlayers) {
        this.minPlayers = minPlayers;
    }

    /**
     * Gets max players.
     *
     * @return the max players
     */
    public int getMaxPlayers() {
        return maxPlayers;
    }

    /**
     * Sets max players.
     *
     * @param maxPlayers the max players
     */
    public void setMaxPlayers(int maxPlayers) {
        this.maxPlayers = maxPlayers;
    }
}
